asapdiscovery-alchemy/asapdiscovery/alchemy/predict.py

```diff
@@ -1046,21 +1048,64 @@ def clean_result_network(network, console=None):
             tf_res = results[0]
 
         deduped_results.append(tf_res)
-
     num_dupes_removed = len(cleaned_results) - len(deduped_results)
+
+    # remove predictions that have a NaN - this is extremely rare
+    denand_results = []
+    for result in deduped_results:
+        if not math.isnan(result.uncertainty.magnitude) and not math.isnan(
```

Comment:
This is now checking the same thing twice.

Reply:
lmao, ok, I'll dig back down just to check what I was initially querying here

Reply:
ok fixed, the intention was to check for NaN in both the prediction and the uncertainty